RestaurantManagementSystem/RestaurantManagementSystem/src/UseCases/Email Confirmation/EmailConfirmationHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using RestaurantManagementSystem.src.Core.Contracts;
+using RestaurantManagementSystem.src.Core.Entities;
+using RestaurantManagementSystem.src.Infrastructure.Services;
+
+namespace RestaurantManagementSystem.src.UseCases.Email_Confirmation
+{
+    public class EmailConfirmationHandler : IEmailConfirmationHandler
+    {
+        private readonly IRestaurantRepo _restaurantRepo;
+        private readonly IEmailService _emailservice;
+        public EmailConfirmationHandler (IRestaurantRepo restaurantRepo,IEmailService emailservice)
+        {
+            _restaurantRepo = restaurantRepo;
+            _emailservice = emailservice;
+        }
+        public async Task<bool> Confirm(ConfirmationCommand confirmationCommand)
+        {
```

Comment:
too much logic here, 
this layer just orchestrates things don't convert to GUID or have the email body defined here 
```